providers/presto/src/airflow/providers/presto/hooks/presto.py

```diff
@@ -152,7 +152,7 @@ def get_conn(self) -> Connection:
 
     def get_isolation_level(self) -> Any:
         """Return an isolation level."""
-        db = self.get_connection(self.presto_conn_id)  # type: ignore[attr-defined]
+        db = self.get_connection(self.presto_conn_id)
```

Comment:
@gopidesupavan +1 for https://github.com/apache/airflow/pull/53252#discussion_r2202912638